test/libsolidity/lsp/publish_diagnostics_3.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.0;
```

Comment:
Does it matter? A change here will require updating all the locations in the expectations...

Reply:
Not so much that it would stop the PR from being merged.

I just think that we won't be keeping them up to date so they will be useless and at most can get in the way in case you want to extract test cases from an older version. The compiler version is implied by the tag on the revision you're taking the code from anyway.